### PR TITLE
User mailer: update email views

### PR DESCRIPTION
**Pull request by davidwessman**
Updating confirmation_instruction and reset_password_instruction

Fixes #71

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -11,10 +11,10 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = 'spindelman@fsektionen.se'
+  config.mailer_sender = 'Röstiga <rostsystem@fsektionen.se>'
```
Comment (navro) [inline config/initializers/devise.rb]: We need to create the alias `rostiga@fsektionen.se` as well :smile:

Comment (davidwessman) [inline config/initializers/devise.rb]: Now it exists 😄 want to use that instead?

Comment (navro) [inline config/initializers/devise.rb]: Yes :+1:

**Comment by davidwessman** [comment]
@navro done

**Comment by navro** [comment]
I don't really know much about mailers, but the code looks better... So +1

**Comment by davidwessman** [comment]
I overridden the Devise mailer, but the views should be cleaner now.

**Pull request merged**